tests/AdagucTests/TestWMS.py

```diff
@@ -231,3 +257,28 @@ def test_WMSGetFeatureInfo_timeseries_forecastreferencetime_json(self):
         AdagucTestTools().writetofile(self.testresultspath + filename,data.getvalue())
         self.assertEqual(status, 0)
         self.assertTrue(AdagucTestTools().compareGetCapabilitiesXML(self.testresultspath + filename, self.expectedoutputsspath + filename))
+
+    def test_WMSGetMap_Report_nounits(self):
+        AdagucTestTools().cleanTempDir()
+        filename="test_WMSGetMap_Report_nounits"
+        reportfilename="./nounits_checker_report.txt"
+        status,data,headers = AdagucTestTools().runADAGUCServer(
+            "source=testdata_report_nounits.nc&service=WMS&request=GetMap&version=1.3.0&layers=sow_a1&crs=EPSG%3A4326&bbox=47.80599631376197%2C1.4162628389784275%2C56.548995855839685%2C9.526486675156528&width=863&height=981&format=image%2Fpng&info_format=application%2Fjson&time=1000-01-01T00%3A00%3A00Z%2F3000-01-01T00%3A00%3A00Z&dim_reference_time=2017-12-15T09%3A00%3A00Z",
```

Comment:
Please move these files inside a folder which explicitly mark them as invalid datasets for testing purposes